pkg/tfgen/docs.go

```diff
@@ -1058,6 +1115,7 @@ func parseArgReferenceSection(subsection []string, ret *entityDocs) {
 			// section is over, but we take it to mean that the current
 			// heading is over.
 			lastMatch = ""
+			bulletListTracker = []bulletListEntry{}
```

Comment:
Unless there is a reason not to use the zero value here (and there shouldn't be), lets use it.
```suggestion
			bulletListTracker = nil
```